examples/basic_cxx_qt_qml/src/mock_qt_types.rs

```diff
@@ -38,6 +39,7 @@ mod mock_qt_types {
                 size: QSize::new(1, 3),
                 sizef: QSizeF::new(1.0, 3.0),
                 time: QTime::new(1, 2, 3, 4),
+                url: Url::from_string("https://github.com/KDAB".to_owned()),
```

Comment:
to_owned() could be removed here if Url::from_string took a `&str`.

Reply:
done